colossalai/shardformer/policies/autopolicy.py

```diff
@@ -104,6 +104,9 @@ class PolicyLocation:
     # Sam
     "transformers.models.sam.modeling_sam.SamModel":
         PolicyLocation(file_name="sam", class_name="SamModelPolicy"),
+    # ChatGLM
+    "tests.kit.model_zoo.transformers.chatglm2_6b.modeling_chatglm.ChatGLMModel":
```

Comment:
You should not register like this, you should register with the normal class name when using chatglm with Transformers.